src/i18n.jl

```diff
@@ -0,0 +1,58 @@
+export AbstractLanguage, set_language!
+abstract type AbstractLanguage end
+
+
+
+
+abstract type English  <: AbstractLanguage end
```

Comment:
I notice that you have the English *_str functions in the same module and file.  
I was originally thinking that it might be nice to have a separate module for each language, so that there's more parallel treatment of different languages.  Is there a disadvantage to that?

Reply:
Even if we end up putting all the *_str functions in one module, I'd suggest that we organize them into separate files.  That way it'll be easier when someone submits a pull request to see that they just added `include("spanish.jl")` to this file and a bunch of stuff in spanish.jl.

Reply:
Done!